src/services/legal-services.ts

```diff
@@ -133,6 +133,30 @@ export default class LegalServices {
       })
   }
```

Comment:
`fetchNameRequest()`, above, should be deleted. This was the whole point of this feature: to get rid of the unsecured NR endpoint.

But I see that fetchNameRequest() is called when the app starts (to validate a name request in a filing) and when the filing is being submitted. I think we won't have the phone or email in those places... but doesn't the new endpoint return the NR data if the affiliation is correct? So maybe these 2 places can use fetchValidContactNr() with empty phone/email and it will work?

Reply:
And actually, I prefer the original name (fetchNameRequest) instead of the new name (fetchValidContactNr), so I propose deleting this function and renaming the one below.

Reply:
Sorry. The new endpoint will not return the NR if the request doesn't include phone or email.

https://github.com/kzdev420/lear/blob/a300f73f65062feb01f1f77cb39787cb5e02b9c7/legal-api/src/legal_api/resources/v2/namerequest.py#L48-L56

We have to include email or phone even if it can be the one blank space ( phone=' '&email=' ' ) for move to affiliate_check process.

Reply:
You don't feel like updating the API to skip the email/phone check if it's correctly affiliated? 😁 

I prefer to not use one blank space, as that's easy to overlook when reading code / JSON.

How about `phone='phone'&email='email'` ?

Reply:
Wait - if you pass nonsense phone and email values, won't this line fail?

https://github.com/kzdev420/lear/blob/a300f73f65062feb01f1f77cb39787cb5e02b9c7/legal-api/src/legal_api/resources/v2/namerequest.py#L73

Reply:
Yes. it will be fail with non-sense email and phone

Reply:
So, we need a solution here, because we definitely want to get rid of the "open" NR endpoint.

Reply:
Can the original `fetchNameRequest` be deleted now?

Reply:
Yes, please remove that function.

This endpoint will have to be removed from Legal API after the UIs are updated. Do you have a plan to do that?

Reply:
Yes, I have a plan.

Reply:
Looks good.